jaxb-ri/core/src/main/java/org/glassfish/jaxb/core/v2/WellKnownNamespace.java

```diff
@@ -23,7 +23,7 @@ private WellKnownNamespace() {} // no instanciation please
 
     public static final String SWA_URI = "http://ws-i.org/profiles/basic/1.1/xsd";
 
-    public static final String XML_MIME_URI = "http://www.w3.org/2005/05/xmlmime";
+    public static final String XML_MIME_URI = "https://www.w3.org/2005/05/xmlmime";
```

Comment:
I believe this shouldn't change. It's the namespace name, which is just a string. And that name is part of the spec.